bin/appliance_console

```diff
@@ -120,6 +121,7 @@ module ApplianceConsole
         summary_entry("#{I18n.t("product.name")} Server", evm_running ? "running" : "not running"),
         summary_entry("#{I18n.t("product.name")} Database", dbhost || "not configured"),
         summary_entry("Database/Region", database ? "#{database} / #{region.to_i}" : "not configured"),
+        summary_entry("Local Messaging Broker", messaging ? "configured" : "not configured"),
```

Comment:
This looks good, think it would be helpful to know if messaging is configured in general in addition to if the broker is running locally:
```suggestion
        summary_entry("Messaging", ManageIQ::ApplianceConsole::MessageConfiguration.configured? ? "configured" : "not configured")
        summary_entry("Local Messaging Broker", messaging ? "configured" : "not configured"),
```